include/klee/Module/KInstruction.h

```diff
@@ -11,15 +11,16 @@
 #define KLEE_KINSTRUCTION_H
 
 #include "klee/Config/Version.h"
-#include "klee/Module/InstructionInfoTable.h"
 
 #include "klee/Support/CompilerWarning.h"
 DISABLE_WARNING_PUSH
 DISABLE_WARNING_DEPRECATED_DECLARATIONS
+#include "KModule.h"
```

Comment:
Warnings should be disabled only for llvm-related headers.

Reply:
Done